plugin.js

```diff
@@ -63,8 +63,15 @@ const buildRouter = (admin, predefinedRouter) => {
           res.send(html)
         }
       } catch (e) {
-        // eslint-disable-next-line no-console
-        console.log(e)
+        if (e.statusCode >= 400 && e.statusCode < 500) {
+          res.status(e.statusCode)
+          res.send({ error: true, status: e.statusCode, notice: e.message })
+        } else {
+          // eslint-disable-next-line no-console
+          console.error(e)
+          res.status(500)
+          res.send({ error: true, status: 500, notice: 'Internal Server Error' })
```

Comment:
I think this change should also work in admin-bro-hapijs. So in case of error there also should be a notice message. But this is not a blocker - we can update it here first and next push the same upadate to hapijs plugin, but please add an issue to admin-bro-hapijs - to not forget about this